packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,141 @@ enum SystemUiOverlay {
+    systemNavigationBarIconBrightness: Brightness.light,
+    statusBarIconBrightness: Brightness.light,
+    statusBarBrightness: Brightness.light,
+  );
+
+  /// Creates a new [SystemUiOverlayStyle].
+  const SystemUiOverlayStyle({
+    this.systemNavigationBarColor,
+    this.systemNavigationBarDividerColor,
+    this.statusBarColor,
+    this.statusBarBrightness,
+    this.statusBarIconBrightness,
+    this.systemNavigationBarIconBrightness,
+  });
+
+  /// The color of the system bottom navigation bar.
+  /// 
+  /// Only honored in Android versions O and greater.
+  final Color systemNavigationBarColor;
+
+  /// The color of the divider between the system's bottom navigation bar and the app's content.
+  /// 
+  /// Only honored in Android versions P and greater.
+  final Color systemNavigationBarDividerColor;
+
+  /// The color of top status bar.
+  /// 
+  /// Only honored in Android version M and greater.
+  final Color statusBarColor;
+
+  /// The brightness of top status bar.
+  /// 
+  /// Only honored in iOS .
+  final Brightness statusBarBrightness;
+
+  /// The brightness of the top status bar icons.
+  /// 
+  /// Only honored in Android version M and greater.
+  final Brightness statusBarIconBrightness;
+
+  /// The brightness of the system navigation bar icons.
+  /// 
+  /// Only honored in Android versions O and greater.
+  final Brightness systemNavigationBarIconBrightness;
+
+  /// Convert this event to a map for serialization.
+  Map<String, dynamic> toMap() {
+    return <String, dynamic>{
+      'systemNavigationBarColor': systemNavigationBarColor?.value,
+      'systemNavigationBarDividerColor': systemNavigationBarDividerColor?.value,
+      'statusBarColor': statusBarColor?.value,
+      'statusBarBrightness': statusBarBrightness?.toString(),
+      'statusBarIconBrightness': statusBarIconBrightness?.toString(),
+      'systemNavigationBarIconBrightness': systemNavigationBarIconBrightness?.toString(),
+    };
+  }
+
+  /// Creates a copy of this theme with the given fields replaced with new values.
+  SystemUiOverlayStyle copyWith({
+    Color systemNavigationBarColor,
+    Color systemNavigationBarDividerColor,
+    Color statusBarColor,
+    Brightness statusBarBrightness,
+    Brightness statusBarIconBrightness,
+    Brightness systemNavigationBarIconBrightness,
+  }) {
+    return new SystemUiOverlayStyle(
+      systemNavigationBarColor: systemNavigationBarColor ?? this.systemNavigationBarColor,
+      systemNavigationBarDividerColor: systemNavigationBarDividerColor ?? this.systemNavigationBarDividerColor,
+      statusBarColor: statusBarColor ?? this.statusBarColor,
+      statusBarIconBrightness: statusBarIconBrightness ?? this.statusBarIconBrightness,
+      statusBarBrightness: statusBarBrightness ?? this.statusBarBrightness,
+      systemNavigationBarIconBrightness: systemNavigationBarIconBrightness ?? this.systemNavigationBarIconBrightness,
+    );
+  }
+
+  @override
+  int get hashCode => 
```

Comment:
We usually reserve `=>` for one-liners that return a value

Reply:
Done